portfolio/src/main/webapp/index.html

```diff
@@ -40,6 +40,8 @@ <h1>Mia Chen</h1>
                 <div id="song-container"></div>
             </div>
         </div>
+        <p>Random Quote of the Day:</p>
+        <p id="string-container"></p>
```

Comment:
nit: "selected-quote-container" or something more descriptive.

Reply:
Thank you for the feedback! I will make sure to work on this by week 4.